AlphaWallet/Tokens/Coordinators/SingleChainTokenCoordinator.swift

```diff
@@ -31,7 +31,7 @@ class SingleChainTokenCoordinator: Coordinator {
     private let cryptoPrice: Subscribable<Double>
     private let assetDefinitionStore: AssetDefinitionStore
     private let eventsDataStore: EventsDataStoreProtocol
-    private let navigationController: UINavigationController
+    private let navigationController: NavigationController
```

Comment:
Seems like we can keep this as `UINavigationController` and to change `TokensCardCoordinator.swift` to use a `UINavigationController` instead.

The `NavigationController` class doesn't seems to be unnecessary anymore. We can probably remove it in a separate PR.